.travis.yml

```diff
@@ -8,6 +8,6 @@ env:
 language: node_js
 
 node_js:
-  - "5"
+  - "6"
```

Comment:
Sure, but the latest not-stable is 0.11 - so it's mostly meaningless. Either way, nvm is going to remove "stable" at some point, so "node" is a safer long-term choice.